src/schema/schema.graphql

```diff
@@ -2353,8 +2359,11 @@ type Mutation {
 	moveTask(inviteId: ID!, destFolderId: ID!): String
 	prefEnableOutOfOfficeAlertOnLogin(value: Boolean!): Boolean
 	prefEnableOutOfOfficeReply(value: Boolean!): Boolean
+	prefEnableOutOfOfficeExternalReply(value: Boolean!): Boolean
```

Comment:
instead of creating separate mutation for individual property changes use modifyPrefs

Reply:
Done